docs/strategies/gradle.md

```diff
@@ -70,9 +74,9 @@ This strategy requires dynamic analysis in its discovery phase (not just in the
 
 When executing Gradle for an analysis target at directory `ANALYSIS_TARGET_DIR`, the CLI will prefer (in order):
 
-1. `$ANALYSIS_TARGET_DIR/gradlew`
-2. `$ANALYSIS_TARGET_DIR/gradlew.bat`
-3. `gradle` (from `$PATH`)
+1. `gradlew`. Search upwards in the directory `ANALYSIS_TARGET_DIR` for the nearest `gradlew` file.
+1. `gradlew.bat`. Search upwards in the directory `ANALYSIS_TARGET_DIR` for the nearest `gradlew.bat` file.
```

Comment:
Phrasing was confusing to me at first.  An alternative: "Recursively search parent directories of `ANALYSIS_TARGET_DIR` until a __ file is found".

Reply:
I changed the wording to `1. gradlew. First looking in ANALYSIS_TARGET_DIR and then recursively searching parent directories until gradlew is found.` so that its clear we look in the ANALYSIS_TARGET_DIR first.